src/lib/relay/config.tsx

```diff
@@ -4,7 +4,7 @@ const { Emission } = NativeModules
 
 let metaphysicsURL
 if (Emission && Emission.useStagingEnvironment) {
-  metaphysicsURL = "https://metaphysics-staging.artsy.net"
+  metaphysicsURL = "http://localhost:5000"
```

Comment:
errr that totally shouldn't have gotten pushed. I was quite specifically adding select things, but perhaps I accidentally `-am`'d after all 😓 Thanks!